libs/note-manager/src/note-manager.ts

```diff
@@ -296,7 +297,7 @@ export class NoteManager {
     this.isSyncingNoteSubject.next(value);
   }
 
-  get defaultNoteGenInput(): DefaultNoteGenInput {
```

Comment:
Why remove this?

Reply:
If we use the `DefaultNoteGenInput`, the `index` property type  `undefined` or `number` but we actually define it as `0` by default, I remove the explicit type for Typescript auto infer the return type.

<table>
<tr>
 <td>
	With `DefaultNoteGenInput`
 <td>
	Without `DefaultNoteGenInput`
<tr>
 <td>
  <img src="https://github.com/webb-tools/webb-dapp/assets/60747384/4d8fb849-0b13-487f-8d03-cbe5da62b744" alt="without DefaultNoteGenInput" />
 <td>
  <img src="https://github.com/webb-tools/webb-dapp/assets/60747384/f85472b8-fdab-4786-bc6c-c62c78324e92" alt="with DefaultNoteGenInput"  />
</table>